components/Buttons/docs/README.md

```diff
@@ -42,11 +42,38 @@ elevation, Material Design ripples, and other stateful design APIs.
 To help ensure your buttons are accessible to as many users as possible, please be sure to review
 the following recommendations:
 
-*  Set an appropriate
+### Set an accessibilityLabel
+Set an appropriate
 [`accessibilityLabel`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel)
 value if your button does not have a title. This is often the case with `MDCFloatingButton`
 instances which typically only have an icon.
-
 ```
 button.accessibilityLabel = @"Create";
-```
\ No newline at end of file
+```
+
+### Minimum touch size
+
+#### Set the frame
+Set your buttons to have a minium size. [Material Touch guidelines](https://material.io/design/layout/spacing-methods.html#touch-click-targets) typically recommend a height and width of 48 points.
+```
+button.minimumSize = CGSizeMake(48, 48);
+```
+
+#### Set the touch size (alternative)
+
+An alternate approach is to set the hit areaInsets to a negative value to make the touch target
+larger than the visual appearance of the button. When you do this you should be careful to make sure
+you pad the distance between the buttons so there is no overlap of the touch targets. This will
+allow your button to have a touch area that is larger than it is visually as show in the
+[layout section of the Material
+spec](https://material.io/design/layout/spacing-methods.html#touch-click-targets).
+
+```
+  CGFloat verticalInset = MIN(0, -(48 - button.frame.size.height) / 2);
+  button .hitAreaInsets = UIEdgeInsetsMake(verticalInset, 0, verticalInset, 0);
+
+```
+
+When you do this make sure to follow the rest of the layout guidence and pad buttons with extra
+space.
```

Comment:
Nit: Let's refer to it as "sufficient" or "recommended" space instead of "extra". Extra sounds like unnecessary.